openapi/index.yaml

```diff
@@ -338,25 +338,204 @@ paths:
+        "401":
+          description: Unauthorized.
+        "429":
+          description: Too many requests.
+        "500":
+          description: The service cannot be created.
+          schema:
+            $ref: "#/definitions/ProblemJson"
+    get:
+      operationId: getUserServices
+      summary: Get User Services
+      description: |
+        Retrieve the identifiers of the services owned by the calling user
+      responses:
+        "200":
+          description: The list of service ids.
+          schema:
+            "$ref": "#/definitions/ServiceIdCollection"
+        "401":
+          description: Unauthorized.
+        "429":
+          description: Too many requests.
+  "/services/{service_id}":
+    parameters:
+      - name: service_id
+        in: path
+        type: string
+        required: true
+        description: The ID of an existing Service.
+    get:
+      operationId: getService
+      summary: Get Service
+      description: |
+        Retrieve a previously created service with the provided service ID.
+        This API operation needs the same API key of the service being retrieved
+        otherwise 403 forbidden will be returned to the caller.
+      responses:
+        "200":
+          description: Service found.
+          schema:
+            "$ref": "#/definitions/ServiceWithSubscriptionKeys"
+        "401":
+          description: Unauthorized.
+        "403":
+          description: Forbidden.
+        "404":
+          description: No service found for the provided ID.
+        "429":
+          description: Too many requests.
+    put:
+      operationId: updateService
+      summary: Update Service
+      description: Update a previously created service with the provided service ID
+      parameters:
+        - in: body
+          name: body
+          description: |-
+            The updated service payload.
+            This API operation needs the same API key of the service being updated
+            otherwise 403 forbidden will be returned to the caller.
+          schema:
+            "$ref": "#/definitions/ServicePayload"
+      responses:
+        "200":
+          description: Service updated.
+          schema:
+            "$ref": "#/definitions/ServiceWithSubscriptionKeys"
+        "401":
+          description: Unauthorized.
+        "403":
+          description: Forbidden.
+        "404":
+          description: No service found for the provided ID.
+        "429":
+          description: Too many requests.
+        "500":
+          description: The service cannot be updated.
+          schema:
+            $ref: "#/definitions/ProblemJson"
+  "/services/{service_id}/logo":
+    parameters:
+      - name: service_id
+        in: path
+        type: string
+        required: true
+        description: The ID of an existing Service.
+    put:
+      summary: Upload service logo.
+      description: |
+        Upsert a logo for an existing service.
```

Comment:
The "same apikey" comment must also be added here.

Reply:
Fixed in [469e173](https://github.com/pagopa/io-functions-services/pull/67/commits/469e173201722102ed04c60b05300f6618a02fe2)